src/java.base/share/classes/java/io/ObjectInputStream.java

```diff
@@ -674,10 +665,10 @@ public Object readUnshared() throws IOException, ClassNotFoundException {
         ClassLoader oldCachedLudcl = null;
         boolean setCached = false; 
```

Comment:
While you're here, please remove the trailing space from line 666.

Reply:
Done.